src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Cognitive Services Content Moderator SDK")]
+[assembly: AssemblyDescription("Provides access to the Microsoft Cognitive Services Content Moderator APIs.")]
+
+[assembly: AssemblyVersion("0.13.0.0")]
```

Comment:
`AssemblyVersion` must be set to the first nuget package version that was published.

Reply:
Checked the first was 0.9.0, so changed AssemblyVersion to 0.9.0.0